src/stepfunctions/steps/integration_resources.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from __future__ import absolute_import
+
+from enum import Enum
+
+"""
+Enum classes for task integration resource arn builder
+"""
+
+
+class IntegrationPattern(Enum):
+    WaitForTaskToken = "waitForTaskToken"
+    WaitForCompletion = "sync"
```

Comment:
Let's model the default "Request Response" pattern here too. The enum value can be the suffix `'.waitForTaskToken'`, `'.sync'`. or `''`

Reply:
Sure. Will update